### PR TITLE
fix: Store QC workflow

### PR DESCRIPTION
**Pull request by ivadym**
### Fixed:
- Store for QC workflow

### Review and tests: 
- [x] Tests pass ✅ 
  - [x] `cg workflow balsamic-qc store <case-id>`
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: BALSAMIC/workflows/PON.smk [hunk 1]
```diff
@@ -84,8 +84,6 @@ if not Path(config["SENTIEON_EXEC"]).exists():
 
 panel_name = os.path.split(target_bed)[1].replace('.bed','')
 
-coverage_references = expand(cnv_dir + "{sample}.{cov}coverage.cnn", sample=config["samples"], cov=['target','antitarget'])
-baited_beds = expand(cnv_dir + "{cov}.bed", cov=['target','antitarget'])
```
Comment (mathiasbio) [inline BALSAMIC/workflows/PON.smk:88]: Ah nice catch, we don't need these as they are inputs to the final rule in the CNVkit pon creation workflow!

**Review by mathiasbio — APPROVED** [review]
Looks good! 🌴 🧙 🔥  Is there anywhere that I can read more about why this fix was required? If there is maybe you can link it here? ❤️

**Comment by ivadym** [comment]
> Looks good! 🌴 🧙 🔥 Is there anywhere that I can read more about why this fix was required? If there is maybe you can link it here? ❤️

This PR is linked to it's respective issue 😜

https://github.com/Clinical-Genomics/BALSAMIC/issues/1285

**Comment by mathiasbio** [comment]
😬 
...
It was a test, you have passed... 🧙

**Pull request merged**